src/scripts/Battle.ts

```diff
@@ -6,7 +6,7 @@
 class Battle {
     static enemyPokemon: KnockoutObservable<BattlePokemon> = ko.observable(null);
 
-    static counter: number = 0;
+    static counter = 0;
```

Comment:
Weird that this is a fix

Reply:
Its automatically assigned by typescript because we are assigning a number

Reply:
Ah makes sense